packages/rocketchat-ui/client/lib/RoomManager.js

```diff
@@ -256,18 +256,19 @@ Tracker.autorun(function() {
 	return connectionWasOnline = connected;
 });
 
-// Reload rooms after login
-let currentUsername = undefined;
-Tracker.autorun(() => {
-	const user = Meteor.user();
-	if ((currentUsername === undefined) && ((user != null ? user.username : undefined) != null)) {
-		currentUsername = user.username;
-		RoomManager.closeAllRooms();
-		return FlowRouter._current.route.callAction(FlowRouter._current);
-	}
-});
+Meteor.startup(() => {
+
+	// Reload rooms after login
+	let currentUsername = undefined;
+	Tracker.autorun(() => {
+		const user = Meteor.user();
+		if ((currentUsername === undefined) && ((user != null ? user.username : undefined) != null)) {
+			currentUsername = user.username;
+			RoomManager.closeAllRooms();
+			FlowRouter.reload();
```

Comment:
I don't know exactly why this code was there.. I only changed the the "function name" not the functionality.

I think this is done because not all places are reactive to user log in, so it need to reload the current route in order to get everything set up.